response_operations_ui/views/collection_exercise.py

```diff
@@ -1104,3 +1106,26 @@ def _add_collection_instrument(short_name, period):
 
     form.formtype.data = ""  # Reset the value on successful submission
     return get_view_sample_ci(short_name, period)
+
+
+@collection_exercise_bp.route("/cir", methods=["GET"])
+@login_required
+def get_cir_service_status():
+    logger.info("Get CIR service status")
+    error_message = None
+    response_content = None
+    try:
+        response_content = cir_controller.get_cir_service_status()
+    except ExternalApiError as e:
+        error_message = (
+            f"Error: {e.error_code.value} "
+            f"{get_error_code_message(e.error_code)} "
+            f"Service: {e.target_service} "
+            f"Cause Exception: {e.__cause__ if e.__cause__ else 'None'}"
+        )
```

Comment:
This error_message wouldn't be this verbose or expose the low level message or cause exception. This is for development and delivery of CIR. The message we return in the user's UI will most likely contain an error code and user-friendly message. That will come from the user stories.